### DISCUSSION TITLE
Pixelation along two edges of 7735 display

### DISCUSSION THREAD

**Discussion by phillmj** [Bodmer/TFT_eSPI#2592] [New processor/display support or capability request] [answered]
Hi, fantastic lib as I'm always telling you!

I'm trying a new 7735 screen found here: [https://wiki.dfrobot.com/0.96_Inch_160_80_Color_SPI_TFT_Display_SKU_DFR0847#target_6](url)

I have tried all the possible definitions of the 7735:

```
/ #define ST7735_INITB
// #define ST7735_GREENTAB
// #define ST7735_GREENTAB2
// #define ST7735_GREENTAB3
// #define ST7735_GREENTAB128    // For 128 x 128 display
#define ST7735_GREENTAB160x80 // For 160 x 80 display (BGR, inverted, 26 offset)
// #define ST7735_ROBOTLCD       // For some RobotLCD arduino shields (128x160, BGR, https://docs.arduino.cc/retired/getting-started-guides/TFT)
// #define ST7735_REDTAB
// #define ST7735_BLACKTAB
// #define ST7735_REDTAB160x80   // For 160 x 80 display with 24 pixel offset
```

All of them exhibit this streaking (one doesn't work at all, and some actually have the screen offset by a lot on two sides). One I'm using is #define ST7735_GREENTAB160x80 

When the display first powers up, the backlight comes on and there's no streaking, so I don't believe it's a faulty display. Also, for some of the above defines, the streaking is different.

Any ideas how to get rid of the streaking? I can't seem to find a way to fix this.

Thanks for your tireless support and amazing libraries. I'm a fan for life.

Mike

![pixelated edges](https://user-images.githubusercontent.com/97892640/232262649-4be1f360-d014-401b-af46-78149598214c.jpeg)

**Comment by PaulVdBo** [comment]
Same problem here... but on opposite sides...

**Comment by phillmj** [comment, marked as answer]
I fixed it.

In the Init and Rotation files, wherever I found a reference to GREENTAB160x80  with colstart and rowstart, I changed it like this:
```
else if (tabcolor == INITR_GREENTAB160x80)
       {
         commandList(Rcmd2green);
         writecommand(TFT_INVON);
         colstart = 24; //26
         rowstart = 0; //1
```
![#3](https://user-images.githubusercontent.com/97892640/232333670-bc4eee84-d97c-45da-bd5f-65eaf6ce8455.jpeg)

**Comment by shashidharybhat** [comment]
Hey @phillmj , I had the same problem but on a 128x160 display. How did you arrive at the rowstart and columnstart values?

**Reply by phillmj** [reply]
Hi, it was a matter of trial and error. You have to edit the tft driver file inside TFT_Drivers.

I believe it was init.h that I edited.

Hth,

Mike

**Comment by Azhilus** [comment]
I am new in this field and I am using a 144x144 display and I am experiencing Kind of a same issue its just that along with pixelation my screen is also shifted in the same manner

![image](https://github.com/Bodmer/TFT_eSPI/assets/66466976/9c9c6c58-9821-4552-8c49-e883419e2d27)
